CHARTER.adoc

```diff
@@ -46,15 +57,15 @@ The TODO Group will host Working Groups for members to contribute to that are in
  ... Meetings will occur at least once per quarter.
  ... Details of meeting mechanisms and instructions for attendance will be published reasonably in advance to all members in good standing.
  .. Communication mechanisms
- ... All public communication not during meetings will take place over the public group mailing list(s) accessible to members and non-members alike.
- ... All private communication not during meetings will take place over the private group mailing list for members only.
+ ... All public communication not during meetings will take place over the public group mailing list(s), real-time chat, and GitHub and will be accessible to members and non-members alike.
+ ... All private communication not during meetings will take place over the private group mailing list, real-time chat, and GitHub and access will be limited to members only.
  .. Level of confidentiality
  ... All communication during meetings will be under+++<u>+++ Chatham House Rules+++</u>++++++<u>+++.+++</u>+++
- ... All communication on the private mailing list will be restricted to members only and not permitted for public distribution.
- ... All communication on the public mailing list will be public.
+ ... All private communication via all media as defined above will be restricted to members only and not permitted for public distribution.
+ ... All communication via all media as defined above will be public.
```

Comment:
Nit-picking: "all communication" would include the private communcation as well. So, as a suggestion:

```suggestion
 ... All non-private communication via all media as defined above will be public.
```

Reply:
@gkunz  The end of the sentence already defines communication will be public. Trying to understand the intent of this sentence - should it be 'All **other** communication via all media as defined above will be public.'?

Reply:
Found it - in TODO original charter it was stated as "All communication on the public mailing list will be public"

Reply:
@tsteenbe, yes, "all other" works well. I just wanted to add a clarifying word because the newly proposed "All communication via all media as defined above will be public." contradicts the statement about the private communication.

Reply:
This does not cover how we use Slack to communicate nor does it clearly define the responsibilities of the SC for communicating about our work to the rest of the group (e.g. what is public, what is private). We should let people know there is private Slack channel for members, there is private Slack channel for the SC. There is nothing nefarious about it, so let's be transparent about how we have evolved to work.

Reply:
@lhawthorn that is somehow documented in the [TODO structure mindmap](https://github.com/todogroup/governance/blob/main/TODO%20Structure/structure-mindmap.md#todo-groups-structure) and in the [community page](https://todogroup.org/community/). 

```
The general members at times share sensitive information and use a private mailing list for that type of discussion. Information shared in this channel is later anonymized and shared with the wider OSPO community via [OSPO Discussions](https://github.com/todogroup/ospology/discussions).

```

Does it make sense to be specific about the communication channels in the charter? Or should we be specific in other documentation (e.g community page) and keep the charter doc more general?